modules/psap-installing-node-feature-discovery-operator.adoc

```diff
@@ -23,14 +23,17 @@ As a cluster administrator, you can install the NFD Operator using the CLI.
 
 . Create a namespace for the NFD Operator.
 
-.. Create the following `Namespace` custom resource (CR) that defines the `openshift-nfd` namespace, and then save the YAML in the `nfd-namespace.yaml` file:
+.. Create the following `Namespace` custom resource (CR) that defines the `openshift-nfd` namespace, and then save the YAML in the `nfd-namespace.yaml` file. Set cluster-monitoring to "true":
```

Comment:
Up to you but I feel like now that the last sentence no longer introduces the YAML file below, you can end it with a period instead of a colon:
```suggestion
.. Create the following `Namespace` custom resource (CR) that defines the `openshift-nfd` namespace, and then save the YAML in the `nfd-namespace.yaml` file. Set cluster-monitoring to "true".
```